mautrix_telegram/portal.py

```diff
@@ -2318,6 +2342,18 @@ async def _handle_telegram_document(
                 "image/": MessageType.IMAGE,
             }.get(info.mimetype[:6], MessageType.FILE),
         )
+        if attrs.is_audio:
+            content["org.matrix.msc1767.file"] = {
+                "url": content.url,
+                "name": content.body,
+                **(content.file.serialize() if content.file else {}),
+                **(content.info.serialize() if content.info else {}),
+            }
+            content["org.matrix.msc1767.audio"] = {"duration": attrs.duration}
```

Comment:
Telegram durations are in seconds

```suggestion
            content["org.matrix.msc1767.audio"] = {"duration": attrs.duration * 1000}
```